apps/designer/app/designer/features/style-panel/shared/parse-css-value.ts

```diff
@@ -1,28 +1,30 @@
+import * as csstree from "css-tree";
 import hyphenate from "hyphenate-style-name";
 import type { StyleProperty, StyleValue, Unit } from "@webstudio-is/css-data";
 import { units } from "@webstudio-is/css-data";
 
-const unitRegex = new RegExp(`${[...units, "number"].join("|")}`);
+const cssTryParseValue = (input: string) => {
+  try {
+    const ast = csstree.parse(input, { context: "value" });
```

Comment:
yes, the problem is this isn't crossbrowser yet, we can replace it some day.
additionally we may need it server-side

Reply:
Essentially only chrome has implemented it atm, because Tab Atkins works at google and probably could make that happen. Neither firefox nor safari have done it yet